<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SstpWidget</class>
 <widget class="QWidget" name="SstpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>441</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_gateway">
          <property name="text">
           <string>Gateway:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_gateway"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_advancedOption">
       <property name="text">
        <string>Advanced...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Optional</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_username">
        <property name="text">
         <string>Username:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_username"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_password">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PasswordField" name="le_password">
        <property name="passwordModeEnabled" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_ntDomain">
        <property name="text">
         <string>NT Domain:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_ntDomain"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_caCert">
        <property name="text">
         <string>CA Certificate:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KUrlRequester" name="kurl_caCert"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="chk_ignoreCertWarnings">
        <property name="text">
         <string>Ignore certificate warnings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>PasswordField</class>
   <extends>QLineEdit</extends>
   <header>passwordfield.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_gateway</tabstop>
  <tabstop>le_username</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>le_ntDomain</tabstop>
  <tabstop>kurl_caCert</tabstop>
  <tabstop>chk_ignoreCertWarnings</tabstop>
  <tabstop>btn_advancedOption</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
